test/datacollector.UnitTests/datacollector.UnitTests.csproj

```diff
@@ -40,11 +40,11 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Xml.XmlDocument">
-      <Version>4.0.1</Version>
+      <Version>4.3.0</Version>
```

Comment:
Just a bit curious, is this upgrade required or is it just done for upgrading to latest version of this package?